docs/csharp/debugger-settings.md

```diff
@@ -174,10 +174,17 @@ Environment variables may be passed to your program using this schema:
 
 The `"console"` setting controls what console (terminal) window the target app is launched into. It can be set to any of these values --
 
-* `internalConsole` - The target process's console output (stdout/stderr) goes to the VS Code Debug Console (default). This is useful for executables that take their input from the network, files, etc. But this does **NOT** work for applications that want to read from the console (ex: `Console.ReadLine`).
-* `integratedTerminal` - The target process will run inside [VS Code's integrated terminal](/docs/editor/integrated-terminal.md). Select the **Terminal** tab in the tab group beneath the editor to interact with your application. Alternatively, add `"internalConsoleOptions": "neverOpen"` to make it so that the default foreground tab is the terminal tab.
+* `"internalConsole"` (default) : the target process's console input (stdin) and output (stdout/stderr) are routed through the VS Code Debug Console. The advantage of this mode is that it allows you to see messages from both the debugger and the target program in one place, so you will not miss important messages or need to switch back and forth. This is useful for programs with simple console interactions (example: using `Console.WriteLine` and/or `Console.ReadLine`). This should NOT be used when the target program needs full control over the console, such as a program that changes the cursor position, uses `Console.ReadKey` for input, etc. See below for instructions on inputting into the console.
+* `"integratedTerminal"` : the target process will run inside [VS Code's integrated terminal](https://code.visualstudio.com/docs/editor/integrated-terminal). Click the 'Terminal' tab in the tab group beneath the editor to interact with your application. When using this mode, by default, the Debug Console will not be shown when starting debugging. This can be configured with `internalConsoleOptions`.
```

Comment:
use relative topic link /docs/editor/integrated-terminal.md
This lets people navigate on github.com and on the vscode-docs staging servers

Reply:
Prefer "Select" to "Click"
Use bold for UI text
So "Select the **Terminal** tab...."